### ISSUE TITLE
var declarations inside functions throw false positives

### ISSUE THREAD

**Issue by rossPatton** [rossPatton/stylint#11]
example:

```
col-width(index)
    first-value = floor((100% / $columns) * index)
    second-value = unit( round( ($gutter-med / ($columns / index) - $gutter-med) - 1, 3), 'px')
    final-value = s('calc(%s + %s)', first-value, second-value)
    return final-value
```

should throw a $ is need error, which is correct

```
col-width($index)
    $first-value = floor((100% / $columns) * $index)
    $second-value = unit( round( ($gutter-med / ($columns / $index) - $gutter-med) - 1, 3), 'px')
    $final-value = s('calc(%s + %s)', first-value, second-value)
    return $final-value
```

should resolve the error, but does not
